ddtrace/bootstrap/sitecustomize.py

```diff
@@ -79,9 +66,12 @@ def update_patched_modules():
 
     opts = {}
 
-    if enabled and enabled.lower() == "false":
-        opts["enabled"] = False
+    if asbool(os.environ.get("DATADOG_TRACE_ENABLED", True)):
```

Comment:
```suggestion
    if asbool(get_env("trace", "enabled", default=True)):
```

?

Reply:
It was intentional to leave it as the functionality here is different than for `DD_TRACE_ENABLED`.